card.py

```diff
@@ -60,6 +60,14 @@ def get_card_points(self):
         Then it returns how many points does the card values according to its name.
         '''
         special_cards_name = self.get_special_cards_name()
+        
         for name in special_cards_name.keys():
-            if name == self.card_name:
-                return special_cards_name[name]
\ No newline at end of file
+            if name == self.name:
+                return special_cards_name[name]
```

Comment:
Add in init name validation and then you can here have just 
```suggestion
    return special_cards_name[name]
```
in place of this loop. Even better if you will use CardType enum defined below :)